docs/sources/flow/getting-started/migrating-from-static.md

```diff
@@ -36,118 +36,116 @@ This topic describes how to:
 
 ## Convert a Grafana Agent Static configuration
 
-To fully migrate Grafana Agent [Static][] to {{< param "PRODUCT_NAME" >}}, you must convert
-your Static configuration into a {{< param "PRODUCT_NAME" >}} configuration.
-This conversion will enable you to take full advantage of the many additional
-features available in {{< param "PRODUCT_NAME" >}}.
+To fully migrate Grafana Agent [Static][] to {{< param "PRODUCT_NAME" >}}, you must convert your Static configuration into a {{< param "PRODUCT_NAME" >}} configuration.
+This conversion will enable you to take full advantage of the many additional features available in {{< param "PRODUCT_NAME" >}}.
 
-> In this task, we will use the [convert][] CLI command to output a {{< param "PRODUCT_NAME" >}}
+> In this task, you will use the [convert][] CLI command to output a {{< param "PRODUCT_NAME" >}}
 > configuration from a Static configuration.
 
-1. Open a terminal window and run the following command:
+1. Open a terminal window and run the following command.
 
    {{< code >}}
 
    ```static-binary
-   AGENT_MODE=flow grafana-agent convert --source-format=static --output=OUTPUT_CONFIG_PATH INPUT_CONFIG_PATH
+   AGENT_MODE=flow grafana-agent convert --source-format=static --output=<OUTPUT_CONFIG_PATH> <INPUT_CONFIG_PATH>
    ```
 
    ```flow-binary
-   grafana-agent-flow convert --source-format=static --output=OUTPUT_CONFIG_PATH INPUT_CONFIG_PATH
+   grafana-agent-flow convert --source-format=static --output=<OUTPUT_CONFIG_PATH> <INPUT_CONFIG_PATH>
    ```
 
    {{< /code >}}
 
    Replace the following:
-    * `INPUT_CONFIG_PATH`: The full path to the [Static][] configuration.
-    * `OUTPUT_CONFIG_PATH`: The full path to output the {{< param "PRODUCT_NAME" >}} configuration.
 
-1. [Start][] {{< param "PRODUCT_NAME" >}} using the new {{< param "PRODUCT_NAME" >}} configuration
-   from `OUTPUT_CONFIG_PATH`:
+    * _`<INPUT_CONFIG_PATH>`_: The full path to the [Static][] configuration.
+    * _`<OUTPUT_CONFIG_PAT>H_`: The full path to output the {{< param "PRODUCT_NAME" >}} configuration.
+
+1. [Start][] {{< param "PRODUCT_NAME" >}} using the new {{< param "PRODUCT_NAME" >}} configuration from _`<OUTPUT_CONFIG_PATH>`_:
 
 ### Debugging
 
-1. If the convert command cannot convert a [Static] configuration, diagnostic
-   information is sent to `stderr`. You can use the `--bypass-errors` flag to
-   bypass any non-critical issues and output the {{< param "PRODUCT_NAME" >}} configuration
-   using a best-effort conversion.
+1. If the convert command cannot convert a [Static] configuration, diagnostic information is sent to `stderr`.
```

Comment:
Not sure why `[Static]` isn't `[Static][]`? It seems to work though.

Reply:
Fixed in the next commit.